steps/run_step.rb

```diff
@@ -4,6 +4,8 @@ module Runner
   class RunStep
     class << self
       def call(cmd)
+        $stdout.sync = true
```

Comment:
I think this is slightly better, because it avoids touching global var directly: 
```suggestion
        STDOUT.sync = true
```